web/src/click_handlers.js

```diff
@@ -399,7 +400,26 @@ export function initialize() {
         const topic_name = $(e.target).attr("data-topic-name");
         message_edit.toggle_resolve_topic(message_id, topic_name);
     });
-
+    document.body.addEventListener("click", (e) => {
```

Comment:
Please look at how we do event listeners in surrounding code and follow this patterns.